bgp/modules/terms.py

```diff
@@ -353,6 +353,32 @@ def __init__(self):
         super().__init__(['copyright', '©'], extractor=self.extractor, match_limit=1)
 
 
+class TocPageDetectorModule(KeywordPageDetectorModule):
+
+    def __init__(self):
+        super().__init__(['contents', 'table of contents', "table of content"], match_limit=1)
```

Comment:
Is there a reason we check both `table of contents` and `table of content`?

For this iteration, maybe we just check `table of content` as a compromise and see how it performs on 100 textbooks.
```suggestion
        super().__init__(["table of content"], match_limit=1)
```

Reply:
I thought that there could be a book which uses 'Table of content' due to a grammatical error or something like that, so for avoiding such edge-cases I included that in the `keywords` list.

Reply:
It's possible; at this stage I think being more strict maybe advantageous and making just see how well it performs with a single rule.